### PR TITLE
fix(DB/Creature): Fix Elder Runetotem & Elder Darkhorn gossip

### PR DESCRIPTION
**Pull request by Logandros**
## Changes Proposed:
-  Fixes the non-working gossip menus for npc 15572 and 15579
-  

## Issues Addressed:

- Closes 

## SOURCE:

https://wotlkdb.com/?npc=15572
https://wotlkdb.com/?npc=15579

## Tests Performed:

-  attempted to use gossip menus on both npcs which were unresponsive and now work as intended
- 

## How to Test the Changes:

1. `.go creature id 15572` and/or `.go creature id 15579`
2. before applying the update the gossip menus do not work
3. after applying the update the gossip menus work

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/elder_gossip_fix.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (6899, 2, 0, 'Where is Elder Wheathoof?', 11582, 1, 1, 6895, 0, 0, 0, '', 0, 0);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (6899, 3, 0, 'Where is Elder Runetotem?', 11588, 1, 1, 21122, 0, 0, 0, '', 0, 0);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (6899, 4, 0, 'Where is Elder Windtotem?', 11576, 1, 1, 21147, 0, 0, 0, '', 0, 0);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/elder_gossip_fix.sql:11]:
```suggestion
DELETE FROM `gossip_menu_option` WHERE `MenuID` IN (6815, 6899);
INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES
(6815, 1, 0, 'Where is Elder High Mountain?', 11578, 1, 1, 21097, 0, 0, 0, '', 0, 0),
(6815, 2, 0, 'Where is Elder Moonwarden?', 11580, 1, 1, 21107, 0, 0, 0, '', 0, 0),
(6815, 3, 0, 'Where is Elder Windtotem?', 11576, 1, 1, 21147, 0, 0, 0, '', 0, 0),
(6815, 4, 0, 'Where is Elder Darkhorn?', 11584, 1, 1, 6896, 0, 0, 0, '', 0, 0),
(6899, 0, 0, 'Where is Elder Darkcore?', 11493, 1, 1, 21086, 0, 0, 0, '', 0, 0),
(6899, 1, 0, 'Where is Elder Ironband?', 11507, 1, 1, 6854, 0, 0, 0, '', 0, 0),
(6899, 2, 0, 'Where is Elder Wheathoof?', 11582, 1, 1, 6895, 0, 0, 0, '', 0, 0),
(6899, 3, 0, 'Where is Elder Runetotem?', 11588, 1, 1, 21122, 0, 0, 0, '', 0, 0),
(6899, 4, 0, 'Where is Elder Windtotem?', 11576, 1, 1, 21147, 0, 0, 0, '', 0, 0);
```

Comment (Logandros) [reply data/sql/updates/pending_db_world/elder_gossip_fix.sql:11]: I had that 2 commits ago with the exception of not combining the 2 deletes, let's see if she works...

**Review by acidmanifesto — APPROVED** [review]

**Comment by Logandros** [comment]
`NOTE: We do not respond well to leechers! If your fix comes from another organization or person you should ALWAYS give credit to the original author and commit.`

That being said, I do want to mention. I did not 100% manually create this fix, it was working correctly on another core so I compared from that to create my sql. I'm sure it is originally someone's work, just covering my humps.. no one likes a leecher!

**Comment by Maelthyrr** [comment]
Tested and work fine for all gossips. Thanks :)

**Pull request merged**